cmd/cloudx/client/api_key.go

```diff
@@ -5,40 +5,100 @@ package client
 
 import (
 	"context"
+	"encoding/json"
 	"errors"
 	"fmt"
+	"io"
+	"net/http"
+	"strings"
 
 	cloud "github.com/ory/client-go"
 	"github.com/ory/x/cmdx"
 )
 
-func (h *CommandHelper) CreateAPIKey(ctx context.Context, projectID, name string) (*cloud.ProjectApiKey, error) {
-	c, err := h.newCloudClient(ctx)
+func (h *CommandHelper) CreateProjectAPIKey(ctx context.Context, projectID, name string) (*cloud.ProjectApiKey, error) {
+	c, err := h.newConsoleAPIClient(ctx)
 	if err != nil {
 		return nil, err
 	}
 
-	token, _, err := c.ProjectAPI.CreateProjectApiKey(ctx, projectID).CreateProjectApiKeyRequest(cloud.CreateProjectApiKeyRequest{Name: name}).Execute()
+	token, res, err := c.ProjectAPI.CreateProjectApiKey(ctx, projectID).CreateProjectApiKeyRequest(cloud.CreateProjectApiKeyRequest{Name: name}).Execute()
 	if err != nil {
+		fmt.Printf("res: %+v\nreq: %+v\n", res, res.Request)
 		return nil, err
 	}
 
 	return token, nil
 }
 
-func (h *CommandHelper) DeleteAPIKey(ctx context.Context, projectIdOrSlug, id string) error {
-	c, err := h.newCloudClient(ctx)
+func (h *CommandHelper) DeleteProjectAPIKey(ctx context.Context, projectID, keyID string) error {
+	c, err := h.newConsoleAPIClient(ctx)
 	if err != nil {
 		return err
 	}
 
-	if _, err := c.ProjectAPI.DeleteProjectApiKey(ctx, projectIdOrSlug, id).Execute(); err != nil {
+	if _, err := c.ProjectAPI.DeleteProjectApiKey(ctx, projectID, keyID).Execute(); err != nil {
 		return err
 	}
 
 	return nil
 }
 
+func (h *CommandHelper) CreateWorkspaceAPIKey(ctx context.Context, workspaceID, name string) (*cloud.WorkspaceApiKey, error) {
+	// TODO replace with SDK method
+	baseURL := CloudConsoleURL("api")
+	c, err := h.newConsoleHTTPClient(ctx)
+	if err != nil {
+		return nil, err
+	}
+	req, err := http.NewRequest(
+		http.MethodPost,
+		fmt.Sprintf("%s/workspaces/%s/tokens", baseURL.String(), workspaceID),
+		strings.NewReader(fmt.Sprintf(`{"name":"%s"}`, name)),
+	)
+	if err != nil {
+		return nil, err
+	}
+	req.Header.Set("Content-Type", "application/json")
+	res, err := c.Do(req)
+	if err != nil {
+		return nil, err
+	}
+	if res.StatusCode != http.StatusCreated {
+		resBody, err := io.ReadAll(res.Body)
+		if err != nil {
+			return nil, err
+		}
+		return nil, fmt.Errorf("expected status code %d but got %d\n%s", http.StatusCreated, res.StatusCode, resBody)
+	}
+	key := cloud.WorkspaceApiKey{}
+	if err := json.NewDecoder(res.Body).Decode(&key); err != nil {
+		return nil, err
+	}
+	return &key, nil
+}
+
+func (h *CommandHelper) DeleteWorkspaceAPIKey(ctx context.Context, workspaceID, keyID string) error {
+	// TODO replace with SDK method
+	baseURL := CloudConsoleURL("api")
+	c, err := h.newConsoleHTTPClient(ctx)
+	if err != nil {
+		return err
+	}
+	req, err := http.NewRequest(http.MethodDelete, fmt.Sprintf("%s/workspaces/%s/tokens/%s", baseURL.String(), workspaceID, keyID), nil)
+	if err != nil {
+		return err
+	}
+	res, err := c.Do(req)
+	if err != nil {
+		return err
+	}
```

Comment:
```suggestion
	}
	}
	defer res.Body.Close()
```

Reply:
Ideally we'll not merge this, but have the SDK published. I'll keep it open as a reminder.